scripts/build-index.js

```diff
@@ -25,12 +25,27 @@ function buildPagesIndex(files) {
       if (!index[page].platform.includes(os)) {
         index[page].platform.push(os);
       }
-
       if (!index[page].language.includes(language)) {
         index[page].language.push(language);
       }
+      let targets = index[page].targets;
+      let needsPush = true;
+      for (const target of targets) {
+        if (target.platform === os && target.language === language) {
+          needsPush = false;
+          continue
+        }
+      }
+      if (needsPush) {
+        targets.push({"os": os, "language": language})
+        index[page].targets = targets;
```

Comment:
```suggestion
        index[page].targets = targets;
```

This line isn't necessary.

Reply:
Yes, I saw that. I wasn't too sure about what JS does internally. Thanks for confirming.